extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php

```diff
@@ -119,6 +119,14 @@ protected function getXmlAttribute( \SimpleXMLElement $xmlNode, $attribute ) {
 			: null;
 	}
 
+	protected function getXmlAttributeFromSupported( \SimpleXMLElement $xmlNode, $attribute, $supportedAttributes ) {
+		$attr = $this->getXmlAttribute( $xmlNode, $attribute );
+		if ( isset($attr) && in_array( $attr, $supportedAttributes ) ) {
+			return $attr;
+		}
+		return self::DEFAULT_TAG_NAME;
```

Comment:
you probably will want static::DEFAULT_TAG_NAME to benefit fully from late static binding